File: Arcade/Intro/C++/19_areEquallyStrong.cpp
bool areEquallyStrong(int yourLeft, int yourRight, int friendsLeft, int friendsRight) {
    int yourMin=min(yourLeft,yourRight);
    int yourMax=max(yourLeft,yourRight);
    int friendsMin=min(friendsLeft,friendsRight);
    int friendsMax=max(friendsLeft,friendsRight);
    return yourMax==friendsMax && yourMin==friendsMin;
}

File: Arcade/Intro/JavaScript/19_areEquallyStrong.js
function areEquallyStrong(yourLeft, yourRight, friendsLeft, friendsRight) {
    
    if (yourLeft+yourRight !== friendsLeft+friendsRight) { return false; }
    
    var you = [yourLeft, yourRight], friend = [friendsLeft, friendsRight];
    you.sort(); friend.sort();
    
    return (you[0] === friend[0]);
}
